.travis.yml

```diff
@@ -65,12 +70,13 @@ rvm:
   - *ruby_2_4
   - *ruby_2_5
   - *ruby_2_6
+  - *ruby_2_7
   - *ruby_head
 
 branches:
   only: 0-10-stable
 
-matrix:
+jobs:
```

Comment:
this an api change in travis?

Reply:
most probably. you may find it yourself by trying the link in my earlier comment